openapi/tests/openapi_integration/test_vector_config.py

```diff
@@ -169,35 +169,37 @@ def create_collection(collection_name, indexing_threshold):
         )
         assert response.ok
         
-    amount_of_vectors = 3000
+    amount_of_vectors = 10
     
     # Collection with indexing enabled
-    create_collection(indexed_name, 1000)
+    create_collection(indexed_name, 10)
     insert_vectors(indexed_name, amount_of_vectors)
     
     # Collection with indexing disabled
-    create_collection(unindexed_name, None)
+    create_collection(unindexed_name, 0)
     insert_vectors(unindexed_name, amount_of_vectors)
     
-    # Get info
+    # Get info indexed
+    sleep(0.5) # wait for indexing to finish
```

Comment:
It is, it becomes more noticeable at larger uploads, it can take a few seconds to complete building the index. I tested this several times, could I have missed something?

Reply:
Good suggestion, done 👍